packages/stg-definitions-v2/src/constant.ts

```diff
@@ -406,6 +414,10 @@ export const ASSETS: Record<TokenName, AssetConfig> = {
                 type: StargateType.Oft,
                 address: '0x0709f39376deee2a2dfc94a58edeb2eb9df012bd',
             },
+            [EndpointId.APE_V2_MAINNET]: {
+                type: StargateType.Oft,
+                address: '0xf1815bd50389c46847f0bda824ec8da914045d14',
```

Comment:
This is incorrect USDT address

```suggestion
                address: '0x674843C06FF83502ddb4D37c2E09C01cdA38cbc8',
```

https://apescan.io/address/0x674843C06FF83502ddb4D37c2E09C01cdA38cbc8#code

Reply:
updated